src/cbc_sdk/platform/users.py

```diff
@@ -251,15 +251,25 @@ def org_urn(self):
         """
         return f"psc:org:{self._cb.credentials.org_key}"
 
-    def grant(self):
+    def grant(self, create=False):
         """
         Locates the access grant for this user.
 
+        Args:
+            create (bool): If True and the user does not already have a grant, attempt to create one.
+
         Returns:
-            Grant: Access grant for this user.
+            Grant: Access grant for this user, or None if the user has none.
         """
         query = self._cb.select(Grant).add_principal(self.urn, self.org_urn)
-        return query.one()
+        try:
+            return query.one()
+        except ObjectNotFoundError:
+            if create:
+                template = {'principal': self.urn, 'org_ref': self.org_urn, 'roles': [],
+                            'principal_name': f"{self.first_name} {self.last_name}", 'profiles': []}
```

Comment:
Does this work? I recall someone mentioning at least roles or profiles was required

Reply:
You know what? I can safely get rid of this. The one place I thought I'd need it, I wound up using a different mechanism. So I can just kill it here.